sigllm/primitives/prompting/anomalies.py

```diff
@@ -116,17 +105,63 @@ def merge_anomaly_seq(anomalies, start_indices, window_size, step_size, beta=0.5
     return np.sort(final_list)
 
 
-def idx2time(sequence, idx_list):
+def idx2time(timestamp, y):
```

Comment:
this is a very simple primitive, I think it can be consolidated with the `timestamp2interval` primitive.

I would also suggest changing the name to something like `format_anomalies` because the goal of the primitive is to take the indices and timestamps and format them into (start, end, score) intervals.